app/utils/database.py

```diff
@@ -1100,10 +1099,11 @@ def init():
     with DATABASE.atomic():
         sh_bootstrap.save()
 
-    ## Create sequence to allow unique ids for digital certificates
-    if type(DATABASE) == PostgresqlDatabase:
+    # Create sequence to allow unique ids for digital certificates
+    if isinstance(DATABASE, PostgresqlDatabase):
+        # pylint: disable=C0301
```

Comment:
Out of curiosity: what is it we are disabling here?

Reply:
Long line (almost all linter disablers can be found with the corresponding linter and code, e.g. `pylint C0301` will have a page explaining this as the first hit).